packages/cli/src/util/dev/server.ts

```diff
@@ -2221,10 +2221,10 @@ export default class DevServer {
     return this.caseSensitive;
   }
 
-  async runDevCommand() {
+  async runDevCommand(restart: boolean = false) {
     const { devCommand, cwd } = this;
 
-    if (devCommand === this.currentDevCommand) {
+    if (devCommand === this.currentDevCommand && !restart) {
```

Comment:
Am I correctly following this PR in that we no longer actively write a refreshed token to disk at all. Instead, we refresh/fetch and store in memory and restart the dev server with the token updated in the env as appropriate?

I can see how this simplifies a lot of things, so just want to make sure I have the plan in mind.

Reply:
Yes, that's right! 👍